style/feedback.less

```diff
@@ -0,0 +1,103 @@
+.feedback {
+	.header {
+		height: @pageHeaderHeight;
+		max-height: @pageHeaderHeight;
+		background-color: #d8d8d8;
+		box-sizing: border-box;
+		padding: 3px 0;
+		font-size: 17px;
+		text-align: center;
+	}
+
+	.body {
+		background-color: #f8f9fa;
+		height: @availableHeight;
+		text-align: center;
+
+		@media @landscape {
+			padding-top: 2px;
+		}
+
+		.textarea-box {
+			padding-top: 3px;
+
+			textarea {
+				width: 228px;
+				height: 120px;
+				border-radius: 2px;
+				background-color: #fff;
+				border: solid 0.5px #000;
+				resize: none;
+
+				@media @landscape {
+					width: 290px;
+					height: 70px;
+				}
+			}
+		}
+
+		.explanation-box {
+			font-size: 14px;
+			font-weight: normal;
+			font-stretch: normal;
+			font-style: normal;
+			line-height: 1.2;
+			letter-spacing: normal;
+			color: #afb3bf;
+			text-align: left;
+			padding-left: 10px;
+			padding-right: 10px;
+
+			a {
+				color: #36c;
+				text-decoration: none;
+
+				&[ nav-selected=true ] {
+					border-radius: 2px;
+					border: solid 1px rgba( 51, 102, 204, 0.48 );
+					background-color: #eaf3ff;
+				}
+			}
+
+			p {
+				margin-top: 4px;
+			}
+
+			@media @landscape {
+				padding-left: 16px;
+				padding-right: 16px;
+
+				p {
+					margin-top: 3px;
+				}
+			}
+		}
+	}
+}
+
+.confirmation-popup {
+	.header {
+		height: @pageHeaderHeight;
+		max-height: @pageHeaderHeight;
+		background-color: #d8d8d8;
+		box-sizing: border-box;
+		padding: 3px 0;
+		font-size: 17px;
+		text-align: center;
+	}
+
+	p.preview-text {
```

Comment:
<img width="178" alt="Annotation 2020-05-20 160931" src="https://user-images.githubusercontent.com/2560096/82456518-9f8f9780-9ab4-11ea-8c18-b52903ce3186.png">

I saw some weird padding on top of the text in my device for both confirm popup